javascript/ql/src/Security/Summaries/PortalEntrySink.qll

```diff
@@ -11,7 +11,7 @@ class PortalEntrySink extends DataFlow::AdditionalSink {
   PortalEntrySink() { this = p.getAnEntryNode(true) }
 
   override predicate isSinkFor(DataFlow::Configuration cfg, DataFlow::FlowLabel lbl) {
-    cfg instanceof TaintTracking::Configuration and
+    cfg = any(DataFlow::Configuration c) and
```

Comment:
any reason for this stylistic change? I think we generally use the `instanceof` operator.

Reply:
That gets flagged as a spurious type test (because `cfg` is declared as a `DataFlow::Configuration`). On the other hand, leaving out the type test entirely gives a warning about not having used `cfg`. I just noticed, though, that I can simply replace the entire predicate body with `any()`, so let me do that instead.

Reply:
Done now.